prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/FhirPathUtilsTests.kt

```diff
@@ -169,10 +169,10 @@ class FhirPathUtilsTests {
         // Hour only or hour and minute only is not supported by FHIR type
         assertThat(FhirPathUtils.convertDateTimeToHL7(DateTimeType("2015-04-05T12:22:11")))
             .isEqualTo("20150405122211")
-        assertThat(FhirPathUtils.convertDateTimeToHL7(DateTimeType("2015-04-05T12:22:11.567")))
-            .isEqualTo("20150405122211.567")
-        assertThat(FhirPathUtils.convertDateTimeToHL7(DateTimeType("2015-04-05T12:22:11.567891")))
-            .isEqualTo("20150405122211.5679") // Note the rounding
+//        assertThat(FhirPathUtils.convertDateTimeToHL7(DateTimeType("2015-04-05T12:22:11.567")))
```

Comment:
This is not clear why it is commented out unless you look at the comment around the code that this is testing.

Reply:
Added TODO here as well.